web/source/osk/browser/subkeyPopup.ts

```diff
@@ -131,8 +126,10 @@ namespace com.keyman.osk.browser {
       let e = this.baseKey;
 
       // And correct its position with respect to that element
+      const _Box = vkbd.element.offsetParent as HTMLDivElement;
+      let rowElement = (e.key as OSKBaseKey).row.element;
       let ss=subKeys.style;
-      var x = dom.Utils.getAbsoluteX(e)+0.5*(e.offsetWidth-subKeys.offsetWidth);
+      var x = e.offsetLeft + (<HTMLElement>e.offsetParent).offsetLeft + 0.5*(e.offsetWidth-subKeys.offsetWidth);
```

Comment:
I worry that this might adversely affect things if the inlined OSK were inserted deep in a page's element hierarchy, which is what the original `getAbsolute(x)` was designed to handle.

Though, I see a relevant CSS change later that I believe remedies this concern.